### DISCUSSION TITLE
arm架构

### DISCUSSION THREAD

**Discussion by xxhk123** [tw93/Pake#280] [Q&A]
后期考虑支持arm64架构linux吗

**Comment by Tlntin** [comment]
指的是pake-cli?还是actions打包好的应用？

**Reply by xxhk123** [reply]
打包好的应用

**Reply by Tlntin** [reply]
这个可以试试，主要是没有测试设备，周末我看看吧。（有一个树莓派4B，但是装的是ubuntu server）

**Reply by liutun** [reply]
同求支持arm64架构的deb包。
现在国内大力推动信创，办公电脑已经换成基于ARM64架构飞腾CPU的银河麒麟系统了，希望能提供打包好的deb包，或者提供详细的编译教程，我们自己编译。谢谢。

**Reply by Tlntin** [reply]
教程的话，按照pake-cli的文档安装就行了。[文档](https://github.com/tw93/Pake/tree/master/bin)
之前有用树莓派试过，arm64打包基本没有问题，但是github actions不支持arm64，用虚拟容器打包的话，速度很慢，所以就没有打包成品了。

**Reply by yizhi108** [reply]
@liutun 请问有没有验证，tuari在麒麟上是否可以正常运行

**Reply by WillieMatthewLiu** [reply]
可以在麒麟运行，但是2.x版本的不行
